.travis.yml

```diff
@@ -143,6 +145,64 @@ jobs:
         - target/cli/linux/conduit install --conduit-version=$CONDUIT_TAG |tee conduit.yml
         - kubectl -n conduit apply -f conduit.yml --prune --selector='conduit.io/control-plane-component'
 
+    # Run integration tests after container images have been published.
+    - stage: integration-test
+
+      language: go
+      go: "1.10.2"
+      go_import_path: github.com/runconduit/conduit
+
+      cache:
+        directories:
+          - vendor
+          - "$HOME/google-cloud-sdk/"
+          - "$HOME/.cache"
+
+      install:
+        - ./bin/dep ensure -vendor-only -v
+        - ./bin/dep status -v
+
+        - |
+          # Install gcloud and kubectl.
+          dir="${CLOUDSDK_INSTALL_DIR:-${HOME}}/google-cloud-sdk"
+          export CLOUDSDK_CORE_DISABLE_PROMPTS=1
+          if [ -d "$dir/bin" ]; then
+              . "$dir/path.bash.inc"
+              gcloud components update
+          else
+              rm -rf "$dir"
+              curl https://sdk.cloud.google.com | bash
+              . "$dir/path.bash.inc"
+          fi
+          gcloud components install kubectl
+        - |
+          # Configure gcloud with a service account.
+          openssl aes-256-cbc -K $encrypted_6af64675f81c_key -iv $encrypted_6af64675f81c_iv -in .gcp.json.enc -out .gcp.json -d
+          gcloud auth activate-service-account --key-file .gcp.json
+          gcloud config set core/project "$GCP_PROJECT"
+          gcloud config set compute/zone "$GCP_ZONE"
+          gcloud config set container/cluster "$GKE_CLUSTER"
```

Comment:
tioli: since all of this cloud setup is repeated elsewhere in travis.yml, you might consider moving it into functions in `bin/_gcp.sh` so they easily be reused and we don't have to have quite so much boilerplate in the travis.yml.

Reply:
Good thinking -- added.